src/components/SidebarSearch/SidebarSearch.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import cx from 'classnames';
+
+import Button from 'Button';
+
+type SidebarSearchProps = {
+  className?: string;
+  id: string;
+  label?: string;
+  onChange: () => void;
+  onClick: () => void;
+  onKeyDown: () => void;
+  onKeyUp: () => void;
+  value: string;
+};
+
+export const SidebarSearch = ({
+  className,
+  id,
+  label,
+  onChange,
+  onClick,
+  onKeyDown,
+  onKeyUp,
+  value
+}: SidebarSearchProps) => {
+  const classNames = cx('cc-sidebar-search', {
+    [className]: className
+  });
+
+  return (
+    <div className={classNames}>
+      <label htmlFor={`search-${id}`} className="cc-sidebar-search__label">
+        {label || 'Search by keyword'}
+      </label>
+      <input
+        className="cc-sidebar-search__input"
+        name={`search-${id}`}
+        onChange={onChange}
+        onKeyDown={onKeyDown}
+        onKeyUp={onKeyUp}
+        placeholder="e.g. fellowship"
```

Comment:
Would it be useful to have the placeholder as a prop? It may not always be "e.g. fellowship".